components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/MonitoringApiServiceImpl.java

```diff
@@ -2027,6 +2029,260 @@ public Response getChildAppsTransportDetails(String managerId, String appName, S
+                        try {
+                            feign.Response workerResponse = WorkerServiceFactory.getWorkerHttpsClient(PROTOCOL +
+                                    generateURLHostPort(worker.getHost(), String.valueOf(
+                                            worker.getPort())), getUsername(), getPassword()).getWorker();
+                            String responseBody = workerResponse.body().toString();
+                            serverHADetails = gson.fromJson(responseBody, ServerHADetails.class);
+                            if (serverHADetails.getClusterId().equals(Constants.NON_CLUSTERS_ID)) {
+                                feign.Response registeredWorkerSiddhiAppsResponse = WorkerServiceFactory
+                                        .getWorkerHttpsClient
+
+                                                (PROTOCOL + generateURLHostPort(worker.getHost(), String.valueOf(
+                                                        worker.getPort())), this.getUsername(), this.getPassword())
+                                        .getAllAppDetails();
+                                if (registeredWorkerSiddhiAppsResponse.status() == 200) {
+                                    Reader inputReader = registeredWorkerSiddhiAppsResponse.body().asReader();
+                                    List<SiddhiAppStatus> totalApps = gson.fromJson(inputReader, new
+                                            TypeToken<List<SiddhiAppStatus>>() {
+                                            }.getType());
+
+                                    for (SiddhiAppStatus siddhiapp : totalApps) {
+                                        SiddhiAppSummaryInfo siddhiAppSummaryInfo = new SiddhiAppSummaryInfo();
+                                        siddhiAppSummaryInfo.setAppName(siddhiapp.getAppName());
+                                        siddhiAppSummaryInfo.setStatus(siddhiapp.getStatus());
+                                        siddhiAppSummaryInfo.setLastUpdate(siddhiapp.getTimeAgo());
+                                        siddhiAppSummaryInfo.setStatEnabled(siddhiapp.isStatEnabled());
+                                        siddhiAppSummaryInfo.setDeployedNodeType("Worker");
+                                        siddhiAppSummaryInfo.setDeployedNodeHost(worker.getHost());
+                                        siddhiAppSummaryInfo.setDeployedNodePort(String.valueOf(worker.getPort()));
+                                        siddhiAppSummaryInfos.add(siddhiAppSummaryInfo);
+                                    }
+                                }
+                            }
+                        } catch (feign.RetryableException ex) {
+                            logger.error("Error ocurred while connecting the node " + worker.getWorkerId());
+                        } catch (IOException e) {
+                            logger.error("error occurred while retrieving response ", e);
```

Comment:
Some of the exceptions were logged only and others have been thrown:
IOException and SQLException both Implemented java.lang.Exception